chcemvediet/settings/common.py

```diff
@@ -35,6 +35,7 @@
     u'django.contrib.sessions',
     u'django.contrib.sites',
     u'django.contrib.messages',
+    u'django.contrib.redirects',
```

Comment:
@martinmacko47 Ostatne url vyzeraju byt OK. Teda nachadzaju sa rozdiely typu, ze: 
url: `/infoziadosti/`, heading: `Zoznam zverejnenych infoziadosti` a title: `Zverejnene ziadosti`.
url:`/pozvanky`, heading: `Posli pozvanku`, a title: `Pozvanky`. 
Ale to si myslim ze je v poriadku, vsak?

Reply:
@viliambalaz `/infoziadosti` aj `/pozvanky` su OK. Su to len skratene nadpisy, ale vyznam je zachovany.